test/runtests.jl

```diff
@@ -132,7 +137,7 @@ end
     for T in [Int64, Int32, Float64, Float32, ComplexF64, ComplexF32]
         @test ComponentArray(a = T[]) == ComponentVector{T}(a = T[])
         @test ComponentArray(a = T[], b = T[]) == ComponentVector{T}(a = T[], b = T[])
-        @test ComponentArray(a = T[], b = (;)) == ComponentVector{T}(a = T[], b = T[])
+        @test_broken ComponentArray(a = T[], b = (;)) == ComponentVector{T}(a = T[], b = T[])
```

Comment:
It's important we don't break this one because the behavior is relied upon in some some large simulation projects. This allows `ComponentArray`s that match a nested model structure to be initialized when one of the internal models doesn't have integrable state.

Reply:
Gotcha, thanks. I'll take a look at it again when I get a chance.

Reply:
Fixed!

Reply:
@jonniedie Another polite ping. :-)